src/pages/Nav/Nav.scss

```diff
@@ -2,24 +2,26 @@
 @use "../../styles/variables" as *;
 
 .nav {
-  position: absolute;
-  width: 12.5rem;
-  border: 0.2rem solid;
-  border-right-color: $color-currency;
+  position: fixed;
+  width: 17%;
```

Comment:
Use rem's here rather than pct, so this becomes an effectively fixed width. With a percentage the display on large terminals goes a bit wrong.
![image](https://user-images.githubusercontent.com/97879313/200856175-138e5fcb-8f90-4d34-b24e-9a5733282398.png)